SparseGrids/Benchmarks/benchDifferentiate.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef _TASMANIAN_BENCHMARK_DIFFERENTIATE_HPP
+#define _TASMANIAN_BENCHMARK_DIFFERENTIATE_HPP
+
+#include "benchCommon.hpp"
+
+bool benchmark_differentiate(std::deque<std::string> &args){
+    if (args.size() < 9) return false;
```

Comment:
Currently, I'm allowed to call `benchmarksgrid differentiate` with >9 arguments (additional anisotropic weights and level limits). Why not have the function return `false` if we don't have exactly 9 arguments? i.e.,
```
if (args.size() != 9) return false;
```
Alternatively, we could add additional info to the help string.

Reply:
Updated the text to indicate that we allow for extra parameters.